### PR TITLE
quick fix: added navigation documentation

### PR DESCRIPTION
**Pull request by ChhikaraBRUH**
Added the documentation of the navigation which I forgot last time.

Browse the preview: https://deploy-preview-9--snkrs-ui.netlify.app/components/navigation/navigation

### CODE REVIEW

File: Components/Navigation/navigation.html [hunk 1]
```diff
@@ -20,5 +20,25 @@
 				<li class="nav-link"><a href="#">GitHub</a></li>
 			</ul>
 		</nav>
+
+		<!-- Navigation Component Usage -->
+		<div class="wrapper">
+			<div class="component-info-div">
+				<h2 class="text-title">Navigation Component</h2>
```
Comment (gadgetfather) [inline Components/Navigation/navigation.html:27]: you can add a border to navigation so that it will be easily differentiable

**Review by RakshaPawar108 — APPROVED** [review]
Looks good to merge.

**Review by gadgetfather — APPROVED** [review]
Well written code 👍

File: Components/Navigation/navigation.html (hunk 1 above)
Comment (ChhikaraBRUH) [reply Components/Navigation/navigation.html:27]: i added box shadow for better distinction

**Pull request merged**